nonsoolmateClient/src/home/components/SideBarTestButton.tsx

```diff
@@ -1,23 +1,25 @@
 import styled from "styled-components";
-import { useNavigate } from "react-router-dom";
 import { commonFlex } from "style/commonStyle";
 import { SideBarTextBoxLayout } from "style/layout/SideBarTextBoxLayout";
-import { TestDisabledIc, RightArrowIc } from "@assets/index";
+import { TestActiveIc, TestDisabledIc, RightArrowIc, RightArrowBlueIc } from "@assets/index";
 
-export default function SideBarTestButton() {
-  const navigate = useNavigate();
+interface SideBarTestProps {
+  handleMoveToHomeTest: Function;
+  goTo: string;
+}
 
-  function handleMoveToHomeTest() {
-    navigate("/home/test");
-  }
+export default function SideBarTestButton(props: SideBarTestProps) {
+  const { handleMoveToHomeTest, goTo } = props;
 
   return (
-    <ButtonBox type="button" onClick={handleMoveToHomeTest}>
-      <TestDisabledIcon />
+    <ButtonBox type="button" onClick={() => handleMoveToHomeTest()}>
+      {goTo === "test" && <TestActiveIcon />}
+      {goTo != "test" && <TestDisabledIcon />}
```

Comment:
```suggestion
      {goTo === "test" ? <TestActiveIcon /> : <TestDisabledIcon />}
```
요러케 삼항연산자를 사용해서 한 줄로 줄여줄 수 있을 거 같아-!!

Reply:
마쟈 삼항 연산자 쓰면 좋을 것 같다!!